pkg/apis/core/validation/shoot_test.go

```diff
@@ -419,6 +419,39 @@ var _ = Describe("Shoot Validation Tests", func() {
 					Expect(ValidateShootHAConfigUpdate(newShoot, shoot)).To(BeEmpty())
 				})
 			})
+
+			Context("Shoot is hibernated", func() {
```

Comment:
Nit, to streamline with other tests 
```suggestion
			Context("shoot is hibernated", func() {
```